### PR TITLE
Updated produce product

### PR DESCRIPTION
**Pull request by Jh00291**
1) Producing a product now specifies a quantity to produce instead of producing 1 each click.

### CODE REVIEW

File: code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java [hunk 1]
```diff
@@ -426,15 +426,34 @@ void editProductManagerOnClick(ActionEvent event) throws IOException {
...
-			this.inventoryVM.produceProduct(this.productsTableView
-					.getSelectionModel().getSelectedItem(), 1);
-		} catch (IllegalArgumentException e) {
-			Alert errorPopup = new Alert(AlertType.ERROR);
-			errorPopup.setContentText(e.getMessage());
-			errorPopup.showAndWait();
-		}
-		this.refreshProductsTableView();
+	    TextInputDialog dialog = new TextInputDialog("1");
+	    dialog.setTitle("Produce Product");
```
Comment (JasonNunez) [inline code/java_project/src/edu/westga/cs3212/inventory_manager/view/InventoryPage.java:430]: Set Magic Strings as constants

**Review by JasonNunez — CHANGES_REQUESTED** [review]
Ensure you're refactoring magics strings and numbers into constants

**Review by JasonNunez — APPROVED** [review]
LGTM!

**Review by jacobevans1 — APPROVED** [review]
Good to go!

**Pull request merged**